src/plugins/kibana/public/discover/partials/no_results.html

```diff
@@ -0,0 +1,51 @@
+<div ng-show="resultState === 'none'" data-test-subj="discoverNoResults">
```

Comment:
I'd prefer if this ng-show was moved to the site where this directive is injected.